Resources/Prototypes/Datasets/Names/borer.yml

```
@@ -1,72 +1,5 @@
-﻿- type: dataset
+﻿- type: localizedDataset
   id: names_borer
```

```suggestion
  id: NamesBorer
```
Ok, since we have a breaking change for forks here anyways I think this is a good time to change the prototype Id to follow naming conventions. Adjust it in the other places accordingly.

---

Fixed in [23025fe](https://github.com/space-wizards/space-station-14/pull/33398/commits/23025fe40e72070b45f8fcdb349360a5a14f4179). Description is also updated as requested.